### DISCUSSION TITLE
Text alignment

### DISCUSSION THREAD

**Discussion by mickwadev** [ShapeCrawler/ShapeCrawler#216] [Q&A]
Hello :)

Is it possible to find out if text from paragraph is centered, aligned left/right etc.?

**Comment by AdamShakhabov** [comment]
Hi :)

Looks like this implementation is not so hard.

Wait this in the next release!

**Reply by mickwadev** [reply]
Cool :) When do You plan next release?

**Reply by AdamShakhabov** [reply]
02/10 (Thursday)

**Reply by AdamShakhabov** [reply]
Alignment support was added in [**v0.28.0**](https://github.com/ShapeCrawler/ShapeCrawler/releases/tag/v0.28.0) release. You can find example of this using **[here](https://github.com/ShapeCrawler/ShapeCrawler/blob/master/samples/ShapeCrawler.Samples/TextExample.cs#L14)**.

_Don't hesitate to report testing issues😉_
